kk/urls_v1.py

```diff
@@ -20,4 +21,5 @@
     url(r'^', include(hearing_comments_router.urls, namespace='v1')),
     url(r'^', include(hearing_child_router.urls, namespace='v1')),
     url(r'^', include(scenario_comments_router.urls, namespace='v1')),
+    url(r'^me', UserDataView.as_view()),
```

Comment:
This should probably be registered in the `router` for all that fancy autodocumentation jazz.